docs/source/installation/for_dev.rst

```diff
@@ -3,8 +3,38 @@ Manim Installation For Developers
 
 	This documentation is for developers who want to contribute to ``ManimCommunity/manim``.
 
-Installing Poetry
-*****************
+
+
+Manim Installation For Developers with pip
+******************************************
+
+If you'd like to contribute to and/or help develop ``manim-community``, you can
+clone this repository to your local device.  To do this, first make sure you
+have ``git`` installed.
```

Comment:
I think `manim-community` should be reworded, the library is called `manim` by now, and nothing in or related to the library is actually called `manim-community`.

```suggestion
If you'd like to contribute to and/or help develop the community maintained
version of ``manim``, you can clone this repository to your local device.  To do this,
first make sure you have ``git`` installed.
```